src/Persistence/MassTransit.DapperIntegration/Configuration/DatabaseProviders.cs

```diff
@@ -0,0 +1,12 @@
+namespace MassTransit.Configuration
+{
+    /// <summary>
+    /// Used internally by the Dapper provider to choose appropriate implementations of support objects.
+    /// </summary>
+    public enum DatabaseProviders
```

Comment:
If in the MassTransit.Configuration namespace, all types for assembly-specific things should be specific. Otherwise, put it in the DapperIntegration namespace.